pages/index.tsx

```diff
@@ -52,6 +54,8 @@ export default function Home() {
           <Link href='/users'>
             <Button appearance='link'>View all users</Button>
           </Link>
+
+          <LocationMap location={new GeoPoint(45.5019, -73.5674)} />
```

Comment:
how would we be able to retrieve a user/groups location once they sign up??

Reply:
Pass .location attribute of Group, Foodbank or Event type into the LocationMap component.